api/v1alpha1/serviceclaim_types.go

```diff
@@ -47,7 +47,7 @@ type ServiceClaimSpec struct {
 
 // ServiceClaimStatus defines the observed state of ServiceClaim
 type ServiceClaimStatus struct {
-	//+kubebuilder:validation:Enum=Pending;Resolved
+	//+kubebuilder:validation:Enum=Pending;Resolved;Invalid
```

Comment:
If the webhook returns an error, the resource is not going to be created. Is that correct?
Then why do we need an `Invalid` state?

Reply:
Please update the docs with this new state: https://github.com/primaza/primaza/blob/main/docs/entities/serviceclaim.md